### DISCUSSION TITLE
现在go-http宣布不再维护了，请问后续有转移到其他机器人的想法吗

### DISCUSSION THREAD

**Discussion by WwwwGaa** [Tsuk1ko/CQ-picfinder-robot#446] [一般讨论]
现在go-http宣布不再维护了，请问后续有转移到其他机器人的想法吗

**Comment by Tsuk1ko** [comment]
只要支持 onebot 11 的都可以用

**Comment by WwwwGaa** [comment]
谢谢你~



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月22日(周三) 晚上11:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Tsuk1ko/cq-picsearcher-bot]现在go-http宣布不再维护了，请问后续有转移到其他机器人的想法吗




 
只要支持 onebot 11 的都可以用
